ui/src/Signin.tsx

```diff
@@ -82,11 +88,13 @@ export class Signin extends PureComponent<Props, State> {
     try {
       await client.users.me()
 
+      this.setState({auth: true})
```

Comment:
would be cooler to have this in redux and have the other components wait for the store to be updated (less implicit relationship on component hierarchy)

Reply:
i completely agree.  i placed this into the bucket of "im not rewriting everything right now".